tests/integration/workflows/nodejs_npm/test_nodejs_npm.py

```diff
@@ -97,7 +99,7 @@ def test_builds_project_with_remote_dependencies(self):
         output_files = set(os.listdir(self.artifacts_dir))
         self.assertEqual(expected_files, output_files)
 
-        expected_modules = {"minimal-request-promise"}
+        expected_modules = {"minimal-request-promise", ".package-lock.json"}
```

Comment:
seems like some tests here were broken and the actual output modules include this `.package-lock.json` file. Fixed those tests

Reply:
That's a garbage package-lock produced by npm7. We can clean that up in the workflow.

Reply:
not sure why locally the integration tests expect it for me, but yeah it seems like it doesn't happen in appveyor